src/interpreter/compiler.ts

```diff
@@ -697,6 +721,11 @@ const compilers: CompilerMapping = {
         ) {
           throw new UnsupportedInitializationError();
         }
+        if (isBuiltinFunctionSymbolTableEntry(entry)) {
```

Comment:
Similarly for this, why don't we check that the entry type is `Variable` and everything else should throw error?

Reply:
Apparently, the entry type can also be `Array` here?